ogr/ogrsf_frmts/lvbag/ogrlvbaglayer.cpp

```diff
@@ -27,17 +27,17 @@ constexpr const size_t nDefaultIdentifierSize = 16;
 
 OGRLVBAGLayer::OGRLVBAGLayer(const char *pszFilename, OGRLayerPool *poPoolIn,
                              char **papszOpenOptions)
-    : OGRAbstractProxiedLayer{poPoolIn},
-      poFeatureDefn{new OGRFeatureDefn{}}, fp{nullptr}, osFilename{pszFilename},
-      eFileDescriptorsState{FD_CLOSED}, oParser{nullptr}, bSchemaOnly{false},
-      bHasReadSchema{false}, bFixInvalidData{CPLFetchBool(
-                                 papszOpenOptions, "AUTOCORRECT_INVALID_DATA",
-                                 false)},
+    : OGRAbstractProxiedLayer{poPoolIn}, poFeatureDefn{new OGRFeatureDefn{}},
```

Comment:
some formatting issue to fix with pre-commit. Cf https://gdal.org/en/stable/development/dev_practices.html#commit-hooks